apps/odk_publish/management/commands/populate_sample_odk_data.py

```diff
@@ -25,27 +26,42 @@ def handle(self, *args, **options):
             if file.is_file():
                 logger.info("Removing file", file=file.name)
                 file.unlink()
+        logger.info("Creating Organization...")
+        organization = odk_publish.Organization.objects.create(name="Caktus Group", slug="caktus")
         logger.info("Creating CentralServers...")
         central_server = odk_publish.CentralServer.objects.create(
-            base_url="https://odk-central.caktustest.net/"
+            base_url="https://odk-central.caktustest.net/",
+            organization=organization,
+        )
```

Comment:
Perhaps we should add any existing users to this organization, and document in the setup instructions that you should sign in before running this command?

I found it a little confusing that my user didn't have access to this organization after going through the setup instructions.